mdsuite/transformations/transformations.py

```diff
@@ -267,30 +271,50 @@ def _save_coordinates(
         -------
         saves the tensor_values to the database_path.
         """
-        try:
-            self.database.add_data(
-                data=data,
-                structure=data_structure,
-                start_index=index + self.offset,
-                batch_size=batch_size,
-                system_tensor=system_tensor,
-                tensor=tensor,
+
+        # turn data into trajectory chunk
+        # data_structure is dict {'/path/to/property':{'indices':irrelevant, 'columns':deduce->deduce n_dims, 'length':n_particles}
+        species_list = list()
+        # data structure only has 1 element
+        key, val = list(data_structure.items())[0]
+        path = str(copy.copy(key))
+        path.rstrip("/")
+        path = path.split("/")
+        prop_name = path[-1]
+        sp_name = path[-2]
+        n_particles = val.get("length", 1)
```

Comment:
this part is a little hard to read altough not difficult. Does it make sense to put it in a function?

Reply:
this is an ugly workaround that I intend to only keep until the transformation refactor. Once transformations use TrajectoryChunkData, this becomes obsolete